plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/GroovySourceFileAllowlist.java

```diff
@@ -152,6 +148,27 @@ private static boolean endsWithIgnoreCase(String value, String suffix) {
         }
     }
 
+    @Extension
+    @SuppressFBWarnings(value = "NP_BOOLEAN_RETURN_NULL", justification = "intentionally not caching negative results")
```

Comment:
Not sure why, but this annotation actually does have to be on the class. If it's on the `isAllowed` itself, SpotBugs still complains.